static/dbview.config.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * @copyright Copyright (C) 2020, Friendica
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * Main view structure configuration file.
+ *
+ * Here are described all the view Friendica needs to work.
+ *
+ * Syntax (braces indicate optionale values):
+ * "<view name>" => [
+ *	"fields" => [
+ *		"<field name>" => ["table", "field"],
+ *		"<field name>" => "SQL expression",
+ *		...
+ *	],
+ *	"query" => "FROM `table` INNER JOIN `other-table` ..."
+ *	],
+ * ],
+ *
+ * If you need to make any change, make sure to increment the DB_UPDATE_VERSION constant value in dbstructure.config.php.
+ *
+ */
+
+return [
+	"tag-view" => [
+		"fields" => ["uri-id" => ["post-tag", "uri-id"],
```

Comment:
Code standards: Please add a line feed after the opening square bracket `"fields" => [`